cmake/FlowConfig.cmake

```diff
@@ -8,6 +8,15 @@ if(NOT DEFINED FDB_VERSION)
     set(FDB_VERSION "5.2.5")
 endif()
 
+if(NOT TLS_DISABLED)
+    message(STATUS Building FDB with TLS bits)
+    set(FDB_BUILD_COMMAND BOOSTDIR=${Boost_INCLUDE_DIRS} $(MAKE) flow fdb_flow fdb_c fdbmonitor FDBLibTLS fdbrpc)
+    set(FDB_SERVER_BUILD_COMMAND make -j8 fdbserver fdbcli)
```

Comment:
Can we also change `FDB_SERVER_BUILD_COMMAND` to use `$(MAKE)` instead of `make -j8`. I believe this issue exists before your patch, its not an issue you introduced. Good to fix it here though.

Reply:
Good catch!

Reply:
df4cd05